app/views/admin/products/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="container bootdey">
+  <div class="col-md-12">
+    <section class="panel">
+      <div class="panel-body">
+        <div class="col-md-6">
+          <div class="pro-img-details">
+            <%= image_tag @product.display_image if @product.image.present? %>
+          </div>
+        </div>
+        <div class="col-md-6">
+          <h4 class="pro-d-title">
+            <strong>
+              <%= t "product.information" %>
+            </strong>
+          </h4>
+          <p>
+            <%= @product.information %>
+          </p>
+          <div class="product_meta">
+            <span class="posted_in"> <strong><%= t "product.category" %>:</strong>
+              <%= @product.category.name %>
```

Comment:
dùng delegate

Reply:
Chị ơi, trong model product em có delegate :name, to: :category, allow_nil: true. 
Khi em gọi product.name nó sẽ ra category của product đó. Nhưng mà k được do trường name nó bị trùng đúng k chị